tests/integration/components/signup-steps/step-one-test.js

```diff
@@ -134,15 +73,12 @@ module('Integration | Component | signup-steps/step-one', function (hooks) {
   });
 
   test('It display error message and disable the button for invalid input', async function (assert) {
```

Comment:
I think this test is not necessary if you have a test to check all the inputs and the submit button in the form. If not, please add those tests to it.

Reply:
I have refactor the tests